src/chi/components/dropdown/dropdown.scss

```diff
@@ -61,6 +61,10 @@
       display: block;
     }
 
+    &-items-wrapper {
+      overflow: overlay;
```

Comment:
@wojciechwlodarczyklumen why not usign auto instead of overlay? It is a legacy value. Also, is it really needed? If we removed it, would it still work?
https://developer.mozilla.org/en-US/docs/Web/CSS/overflow